include/toml++/impl/std_algorithm.h

```diff
@@ -0,0 +1,12 @@
+//# This file is a part of toml++ and is subject to the the terms of the MIT license.
+//# Copyright (c) Mark Gillard <mark.gillard@outlook.com.au>
+//# See https://github.com/marzer/tomlplusplus/blob/master/LICENSE for the full license text.
+// SPDX-License-Identifier: MIT
+#pragma once
+
+#include "preprocessor.h"
+TOML_DISABLE_WARNINGS;
+#if !TOML_HAS_CUSTOM_OPTIONAL_TYPE
+#include <algorithm>
```

Comment:
See my note about `<algorithm>` earlier.

(Also I think the #if check for `TOML_HAS_CUSTOM_OPTIONAL_TYPE` is a mistake; copy paste?)